android/app/src/main/java/com/chorestory/app/ParentQuestDetailsActivity.java

```diff
@@ -144,28 +156,29 @@ public void onTimeSet(TimePicker view, int hourOfDay, int minute) {
             }
         });
 
-        // TODO: Recurrence Spinner
-
         cancelButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
+                disableButtons();
                 onBackPressed();
             }
         });
 
-        // TODO: send modify quest request
         saveButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
+                disableButtons();
```

Comment:
I thought we were using `disableViews()` @jimzhaozhengwang

Reply:
jim said
"this is correct, ChoreStoryActivity has disableButtons
while ChoreStoryFragment has disableViews"
so im gonna merge this

Reply:
`Activities` inherit from `ChoreStoryActivity` and should call `disableButtons`

`Fragments` inherit from `ChoreStoryFragment` and should call `disableViews`